csharp/src/ProtocolBuffers/FieldAccess/FieldAccessorTable.cs

```diff
@@ -95,5 +106,17 @@ public FieldAccessorTable(Type type, MessageDescriptor descriptor, string[] prop
                 return accessors[field.Index];
             }
         }
+
+        public OneofAccessor this[OneofDescriptor oneof]

```

Comment:
This one is not used anywhere, how about removing it for now and just addressing the above TODO later?